backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/badge/BpkBadge.kt

```diff
@@ -60,40 +63,56 @@ fun BpkBadge(
     Row(
```

Comment:
Commenting here because I can't comment on the `BpkBadge` composable function itself.
This doesn't have anything to do with your PR, but while we're at it can you set a default value for `type` (should be `Normal`). then we can also update the `default` test to not set a type - it confused me a lot that the default snapshot had a destructive case.

Reply:
Thanks for pointing this out, it should now be fixed!